modules/exploits/netware/smb/lsass_cifs.rb

```diff
@@ -84,8 +88,8 @@ def exploit
       NDR.long(0x000f0fff)
 
     resp = dcerpc.call(0x2c, stb)
-    handle, = resp[0,20]
-    code, = resp[20, 4].unpack('V')
+    handle, = resp[0, 20]
+    resp[20, 4].unpack('V')
```

Comment:
```suggestion
    _code, *_other = resp[20, 4].unpack('V')
```

I assume this is useful for debugging? 🤷

Reply:
Re-added

Reply:
Looks like both have ended up here now:

```suggestion
```

And I think the previous line might have changed semantics:

```ruby
resp = (0..100).to_a
handle_a, = resp[0,20]
handle_b = resp[0,20]

puts "handle_a=#{handle_a.inspect}"
puts "handle_b=#{handle_b.inspect}"
```

```ruby
handle_a=0
handle_b=[0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]
```

Reply:
Fixed